src/Moryx.ControlSystem.ProcessEngine/Facades/ProcessControlFacade.cs

```diff
@@ -57,12 +57,18 @@ public void Deactivate()
             ActivityPool.ActivityUpdated -= ParallelOperations.RemoveListener<ActivityUpdatedEventArgs>(OnActivityChanged);
         }
 
-        public IReadOnlyList<IProcess> GetProcesses(ProductInstance productInstance)
+        public Task<IReadOnlyList<IProcess>> GetArchivedProcesses(ProductInstance productInstance)
         {
             ValidateHealthState();
             return ProcessArchive.GetProcesses(productInstance);
         }
 
+        public IAsyncEnumerable<IProcessChunk> GetArchivedProcesses(ProcessRequestFilter filterType, DateTime start, DateTime end, long[] jobIds)
```

Comment:
Do we need/want to have the `IProcessChunk` as a an additional business model on the facade? I see the job based grouping has some value, but I I want to have processes for jobs I could also request the jobs on the `JobManagementFacade` 🤔